libV2/constants.js

```diff
@@ -1,3 +1,14 @@
+
+const truthy = new Set([
+    'true',
+    'on',
+    'yes',
+    'y',
```

Comment:
I've never seen `y` and `t` used in practice in our product, but why not

Reply:
the python stdlib has a method similar to this https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool

Reply:
lolthats where the list originally came from